webapp/channels/src/types/store/views.ts

```diff
@@ -23,6 +24,10 @@ export type AdminConsoleUserManagementTableProperties = {
     sortColumn: string;
     sortIsDescending: boolean;
     pageSize: number;
+    pageIndex: number;
+    userId: string;
```

Comment:
`cursorStartAtUserId`

Reply:
I went with `cursorUserId`.
As a note, I'm usually pretty adverse to smurf-naming, especially since I think you still need additional context to understand why these values are part of the cursor, but hopefully this helps to find that answer at least.